assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1222,7 +1233,7 @@ describe("DiversionPage", () => {
      * components, and we want to make sure that we don't accidentally
      * render it when we didn't intend to.
      */
-    test("the button doesn't render yet in context", async () => {
+    test("when not in test group, is not present", async () => {
```

Comment:
I think the comment above this can go away now.

Reply:
https://github.com/mbta/skate/compare/67e118cb595b85d64dd29f50cd987ca1a0776045..a0147d026bc04ce11131c677e0e07c2f18d9bc0b